React-Basic-Video-Chat/package.json

```diff
@@ -8,7 +8,12 @@
     "opentok-react": "^0.7.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "react-scripts": "1.0.16"
+    "react-scripts": "1.0.16",
```

Comment:
@austinwu13 , would something like update `react-scripts` by itself work?  Something like:
```
"react-scripts: "^1.0.16"
```

Please advise.  Thanks!

Reply:
Hm I tried this and it does not seem to work.